packages/react-components/src/localization/locales/ar-SA/strings.json

```diff
@@ -74,12 +77,16 @@
     "rttCancelButtonLabel": "إلغاء الأمر",
     "rttCloseModalButtonAriaLabel": "إغلاق مربع حوار عرض الكتابة في الوقت الحقيقي المشروط"
   },
+  "captionsBanner": {
+    "captionsBannerSpinnerText": "يتم الآن بدء التسميات التوضيحية..."
+  },
   "mentionPopover": {
     "mentionPopoverHeader": "الاقتراحات"
   },
   "imageOverlay": {
     "downloadButtonLabel": "تنزيل",
-    "dismissButtonAriaLabel": "إغلاق"
+    "dismissButtonAriaLabel": "إغلاق",
+    "overlayVisibleMessage": "Image Details Overlay"

```

Comment:
WHAT IS THIS GUY?

Reply:
Good catch. That string was merged in an hour ago. Let me remove it.